Makefile.js

```diff
@@ -708,6 +736,51 @@ target.checkLicenses = function() {
     });
 };
 
+target.checkGitCommit = function() {
+    var commitMsgs,
+        failed;
+
+    if (hasBranch("master")) {
```

Comment:
Seems to work in my [test PR](https://github.com/eslint/eslint/pull/3096) from my fork with an old local master: 

- Single Commit:  
  - Travis: https://travis-ci.org/eslint/eslint/jobs/71854295
  - Appveyor: https://ci.appveyor.com/project/nzakas/eslint/build/312/job/vawgck8fah0qbtm3
- Multiple Commits: 
  - Travis: https://travis-ci.org/eslint/eslint/jobs/71855115
  - Appveyor: https://ci.appveyor.com/project/nzakas/eslint/build/313/job/y2ax26siw4ymghg4

Is this a sufficient test case, or am I still missing something?